avocado/utils/multipath.py

```diff
@@ -161,6 +161,36 @@ def get_path_status(disk_path):
                     return(paths['dm_st'], paths['dev_st'], paths['chk_st'])
 
 
+def fail_path(path):
+    """
+    failing the individual paths
+    :param disk_path: disk path. Example: sda, sdb.
+    :return: True or False
+    """
+    cmd = 'multipathd -k"fail path %s"' % path
+    if process.system(cmd) == 0:
+        time.sleep(2)
```

Comment:
Please avoid giving random sleep values. A uniform value of sleep everywhere would be nice.

Reply:
Actually the best approach here would be a helper function that returns True/False and a call to `avocado.utils.wait.wait_for()` with a timeout, passing that function. See comment here with an example:
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/605#discussion_r133906038